app/Resources/translations/messages.pt_BR.xlf

```diff
@@ -273,6 +289,15 @@
                 <target>Post excluído com sucesso!</target>
             </trans-unit>
 
+            <trans-unit id="notification.comment_created">
+                <source>notification.comment_created</source>
+                <target>Seu post recebeu um comentário!</target>
+            </trans-unit>
+            <trans-unit id="notification.comment_created.description">
+                <source>notification.comment_created.description</source>
+                <target><![CDATA[Seu post "%title%" recebey um novo comentário. Você pode ler o comentário através <a href="%link%">deste link</a>]]></target>
```

Comment:
Hey Javier.. 

I'm glad you found that as it is a typo. It was supposed to be _recebeu_ (third-person singular). The one you suggested is used for first-person singular.